### PR TITLE
chore: added dkim field

### PR DESCRIPTION
**Pull request by aferraromedina**
Fixes:  Add a property for DKIM in system-usage endpoint  Jira Ticket, ie. [DW-1321](https://makingsense.atlassian.net/browse/DW-1321)

### Checklist:

- [x] I have paid attention to this PR title and description
- [x] I have performed a self-review of my code
- [x] I have built it locally (or my changes does not affect the build)
- [x] I have checked all tests still run ok at Doppler.ReportingApiTest project
- [x] I have added at least one simple unit test covering the new code

### CODE REVIEW

**Comment by andresmoschini** [comment]
👍 

@MarianoHerrera could you take a look?

File: Doppler.ReportingApi/Infrastructure/SummaryRepository.cs [hunk 1]
```diff
@@ -83,11 +83,26 @@ public async Task<SystemUsageSummary> GetSystemUsageAsync(string accountName)
...
                 SELECT
-                    HasCampaignCreated AS HasCampaingsCreated,
-                    HasListCreated AS HasListsCreated,
-                    HasCampaignSent AS HasCampaingsSent
-                FROM dbo.[User]
-                WHERE Email = @accountName";
+                    HasCampaignCreated,
+                    HasListCreated,
+                    HasCampaignSent,
+                    ISNULL(
```
Comment (MarianoHerrera) [inline Doppler.ReportingApi/Infrastructure/SummaryRepository.cs]: Esto mismo se consigue utilizando el outer apply, en el mismo haces el top 1 del domaininformationxuser del usuario encontrado, lo cual va a quedar mucho mas claro.
Avisame cuando tengas 5 min si queres y lo vemos juntos

Comment (aferraromedina) [reply Doppler.ReportingApi/Infrastructure/SummaryRepository.cs]: image
Algo asi seria?

Comment (MarianoHerrera) [reply Doppler.ReportingApi/Infrastructure/SummaryRepository.cs]: Correcto, asi la parte del select queda mas legible. Buen trabajo.

**Pull request merged**